virtualization/Docker/scripts/aiocoap

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+# Installs a modified coap client with support for dtls for use with IKEA Tradfri
+
+# Stop on errors
+set -e
+
+python3 -m pip install cython
+
+cd /usr/src/app/
+mkdir -p build && cd build
+
+git clone --depth 1 https://git.fslab.de/jkonra2m/tinydtls
+cd tinydtls
+autoreconf
+./configure --with-ecc --without-debug
+cd cython
+python3 setup.py install
+
+cd ../..
+git clone --depth 1 https://github.com/chrysn/aiocoap/
+cd aiocoap
+git reset --hard 0df6a1e44582de99ae944b6a7536d08e2a612e8f
```

Comment:
Maybe a silly question: why are we cloning with depth 1 and bounding to specific commit? Isn't it going to fail very often (with every new commit in aiocoap)?

Reply:
Good spot, you're correct. I'll raise a PR to fix.